pre-registration/pre-registration-application-service/src/main/resources/application.properties

```diff
@@ -1,12 +1,13 @@
 spring.application.name=pre-registration-application-service
-spring.profiles.active=local
+spring.profiles.active=mz
 management.endpoint.health.show-details=always
 management.endpoints.web.exposure.include=info,health,refresh
 
 server.servlet.context-path=/preregistration/v1
 
 #server.port=9092
-
+spring.security.user.name=root
+spring.security.user.password=s3cr3t
```

Comment:
no password is allowed in clear text. Choose a new password and store it as encrypted ASAP.